libdebug/ptrace/ptrace_interface.py

```diff
@@ -238,6 +238,12 @@ def cont(self: PtraceInterface) -> None:
         else:
             self._global_state.handle_syscall_enabled = False
 
+        # Reset the event type
+        self._internal_debugger.resume_context.event_type = None
+
+        # Reset the breakpoint hit
```

Comment:
Shouldn't this reset be performed for every control flow instruction and not just cont?